CHANGELOG.md

```diff
@@ -14,8 +14,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * [#79](https://github.com/nf-core/atacseq/issues/79) - Parallelize DESeq2
 * [#80](https://github.com/nf-core/atacseq/pull/80) - Added social preview image
 * [#97](https://github.com/nf-core/atacseq/issues/97) - PBC1, PBC2 from pipeline?
+* [nf-core/chipseq#153](https://github.com/nf-core/chipseq/issues/153) - Add plotHeatmap
```

Comment:
Is it intentional that this stuff here ends up in `atacseq` ?

Reply:
Yep 👍 The pipelines share commonality in quite a few processes so I tend to update them both at the same time to fix bugs etc.

Reply:
PS: I did bump versions but reverted back to the `dev` version to be able to test on AWS because of that issue with the release container not being built when pushing to `master`. Will bump-versions once I have tested this though 👍